### PR TITLE
Fix recording just made not playing back if reload it

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
**Testing**

Setup:
- Run the PR Interface and Sandbox.
- Run playRecordingAC.js from the attached record.zip as a Sandbox script.
- Run record.js from the attached record.zip as an Interface script.

Using the "Recording" app in the toolbar / tablet:
- Make a recording. The recording should automatically play.
- Without restarting Interface or Sandbox, stop the recording just made then load it again. It should play.
- Restart Interface and Sandbox. The recording should play.
- Stop the recording, restart Interface and Sandbox, reload the recording. The recording should play.

[record.zip](https://github.com/highfidelity/hifi/files/1319604/record.zip)

**Comment by YuppyPlays** [comment]
Awaiting build for QA.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3503/HighFidelity-Beta-PR11424-4ce5e4ac46c5b2b4e5cdeb13a55f4ba773ac52cf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3503/HighFidelity-Beta-PR11424-4ce5e4ac46c5b2b4e5cdeb13a55f4ba773ac52cf.exe

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete:

Setup:
Run the PR Interface and Sandbox. **Check**
Run playRecordingAC.js from the attached record.zip as a Sandbox script. **Check**
Run record.js from the attached record.zip as an Interface script. **Check**

Using the "Recording" app in the toolbar / tablet:
Make a recording. The recording should automatically play. **Check**
Without restarting Interface or Sandbox, stop the recording just made then load it again. It should play. **Check**
Restart Interface and Sandbox. The recording should play. **Check**
Stop the recording, restart Interface and Sandbox, reload the recording. The recording should play. **Check**

All looks good to me **Pass** :) not a single issue encountered.

File: libraries/script-engine/src/RecordingScriptingInterface.cpp [hunk 1]
```diff
@@ -54,9 +54,24 @@ float RecordingScriptingInterface::playerLength() const {
     return _player->length();
 }
 
+void RecordingScriptingInterface::playClip(NetworkClipLoaderPointer clipLoader, const QString& url, QScriptValue callback) {
+    _player->queueClip(clipLoader->getClip());
+
+    if (callback.isFunction()) {
+        QScriptValueList args{ true, url };
```
Comment (kencooke) [inline libraries/script-engine/src/RecordingScriptingInterface.cpp]: space before {

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3529/HighFidelity-Beta-PR11424-a2d8152a3a8031a810fd445038d7b9d8fe5ae3f5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3529/HighFidelity-Beta-PR11424-a2d8152a3a8031a810fd445038d7b9d8fe5ae3f5.exe

**Pull request merged**